govc/USAGE.md

```diff
@@ -4373,6 +4467,79 @@ Examples:
 Options:
 ```
 
+## namespace.vmclass.change
+
+```
+Usage: govc namespace.vmclass.change [OPTIONS]
+
+Modifies an existing virtual machine class. 
+
+Examples:
+  govc namespace.vmclass.change -id=test-class-01 -cpus=8 -memory=8192"
+
+Options:
+  -cpus=0                The number of CPUs.
+  -id=                   The identifier of the virtual machine class.
+  -memory=0              The amount of memory (in MB).
+```
+
+## namespace.vmclass.create
+
+```
+Usage: govc namespace.vmclass.create [OPTIONS]
+
+Creates a new virtual machine class. 
+
+Examples:
+  govc namespace.vmclass.create -id=test-class-01 -cpus=8 -memory=8192"
+
+Options:
+  -cpus=0                The number of CPUs.
+  -id=                   The identifier of the virtual machine class.
```

Comment:
There are constraints on the id, not sure if that is the place to document, but it sounds like it? https://opengrok2.eng.vmware.com/xref/main.perforce.1666/bora/vpx/wcp/wcpsvc/vmodl/namespace_management/VirtualMachineClasses.vmodl?r=12697864#156
```
/**
[159 ](https://opengrok2.eng.vmware.com/xref/main.perforce.1666/bora/vpx/wcp/wcpsvc/vmodl/namespace_management/VirtualMachineClasses.vmodl?r=12697864#159)    * Identifier of the virtual machine class. This has DNS_LABEL restrictions
[160 ](https://opengrok2.eng.vmware.com/xref/main.perforce.1666/bora/vpx/wcp/wcpsvc/vmodl/namespace_management/VirtualMachineClasses.vmodl?r=12697864#160)    * as specified in <a href="https://tools.ietf.org/html/rfc1123"></a>. This
[161 ](https://opengrok2.eng.vmware.com/xref/main.perforce.1666/bora/vpx/wcp/wcpsvc/vmodl/namespace_management/VirtualMachineClasses.vmodl?r=12697864#161)    * must be an alphanumeric (a-z and 0-9) string and with maximum length
[162 ](https://opengrok2.eng.vmware.com/xref/main.perforce.1666/bora/vpx/wcp/wcpsvc/vmodl/namespace_management/VirtualMachineClasses.vmodl?r=12697864#162)    * of 63 characters and with the '-' character allowed anywhere except
[163 ](https://opengrok2.eng.vmware.com/xref/main.perforce.1666/bora/vpx/wcp/wcpsvc/vmodl/namespace_management/VirtualMachineClasses.vmodl?r=12697864#163)    * the first or last character. This name is unique in this vCenter server.
[164 ](https://opengrok2.eng.vmware.com/xref/main.perforce.1666/bora/vpx/wcp/wcpsvc/vmodl/namespace_management/VirtualMachineClasses.vmodl?r=12697864#164)    */
```